### PR TITLE
Improvements to workflow subcommand

### LINKED ISSUE #33: Improve design for `relay workflow download`
Currently `relay workflow download <name>` simply outputs the raw yaml to stdout. The code comments and review on #31 suggest this isn't ideal and needs some further design/usability exploration.

### PR DESCRIPTION
**Pull request by ahpook**
Adds a `-f` option to `relay workflow download` for specifying
output file, to match the other `relay workflow *` commands

Adds a shortcut `-p` for `--parameter` on `relay workflow run`

Closes #33

### CODE REVIEW

File: pkg/cmd/workflow.go [hunk 1]
```diff
@@ -248,7 +248,7 @@ func newRunWorkflowCommand() *cobra.Command {
 		RunE:  doRunWorkflow,
 	}
 
-	cmd.Flags().StringArray("parameter", []string{}, "Parameters to invoke this workflow run with")
+	cmd.Flags().StringArrayP("parameter", "p", []string{}, "Parameters to invoke this workflow run with")
```
Comment (bradhe) [inline pkg/cmd/workflow.go:251]: This is good, we should standardize on adding shortcuts. Let's do a quick check to make sure we're doing this everywhere?

File: pkg/cmd/workflow.go [hunk 2]
```diff
@@ -263,13 +263,23 @@ func doDownloadWorkflow(cmd *cobra.Command, args []string) error {
...
+		Dialog.WriteString(body)
+	} else {
+		// make a byte array out of the body to feed to the writer
+		contents := []byte(body)
+		err := ioutil.WriteFile(filepath, contents, 0644)
+		if err != nil {
+			debug.Logf("failed to write to file %s: %s", filepath, err.Error())
+			return err
+		}
+	}
```
Comment (bradhe) [inline pkg/cmd/workflow.go:282]: The commentary here is a bit worthless :) We should remove both of them. You can also make it a bit more terse.

```suggestion
	if filepath == "" {
		Dialog.WriteString(body)
	} else {
		if err := ioutil.WriteFile(filepath, []byte(body), 0644); err != nil {
			debug.Logf("failed to write to file %s: %s", filepath, err.Error())
			return err
		}
	}
```

File: pkg/cmd/workflow.go (hunk 1 above)
Comment (bradhe) [reply pkg/cmd/workflow.go:251]: I did a quick check and everything looks good--we're now using short hand for flags everywhere with this change.

Comment (ahpook) [reply pkg/cmd/workflow.go:251]: 👍

**Pull request merged**